Frederik.tex

```diff
@@ -43,7 +43,7 @@
 sourced? They ported the browser to use Qt for the UI, with seven people in an
 amazing sprint of five days, creating "QtScape" as a proof of concept.
 
-On the people front, over time many a KDE developer started to work on Qt. Yours
+On the people front, over time many KDE developers started to work on Qt. Yours
```

Comment:
Not a native speaker here, clearly my english is not good enough :)
I'll drop this change.